lms/djangoapps/mobile_api/utils.py

```diff
@@ -43,13 +44,6 @@ def mobile_view(is_user=False):
     """
     Function and class decorator that abstracts the authentication and permission checks for mobile api views.
     """
-    class IsUser(permissions.BasePermission):
```

Comment:
@gwprice These are the changes we talked about to share the "IsUser" code. See this file, the test right above, and openedx.core.lib.api.permissions.

If you are OK with me IsUserInUrlOrStaff, let me know. To be conservative, I kept the behavior for mobile as it was.